src/ansys/additive/core/microstructure.py

```diff
@@ -42,43 +42,60 @@ class MicrostructureInput:
     Units are SI (m, kg, s, K) unless otherwise noted.
     """
 
-    #: Default minimum x, y, z, position coordinate (m).
     DEFAULT_POSITION_COORDINATE = 0
-    __MIN_POSITION_COORDINATE = 0
-    __MAX_POSITION_COORDINATE = 10
-    #: Default sample size (m) in each dimension.
+    """Default x, y, z, position coordinate (m)."""
+    MIN_POSITION_COORDINATE = 0
+    """Minimum x, y, z, position coordinate (m)."""
+    MAX_POSITION_COORDINATE = 10
+    """Maximum x, y, z, position coordinate (m)."""
     DEFAULT_SAMPLE_SIZE = 1.5e-3
-    __MIN_SAMPLE_SIZE = 0.001
-    __MAX_SAMPLE_SIZE = 0.01
-    #: Default sensor dimension (m).
+    """Default sample size (m) in each dimension."""
+    MIN_SAMPLE_SIZE = 0.001
+    """Minimum sample size (m) in each dimension."""
+    MAX_SAMPLE_SIZE = 0.01
+    """Maximum sample size (m) in each dimension."""
     DEFAULT_SENSOR_DIMENSION = 5e-4
-    __MIN_SENSOR_DIMENSION = 1e-4
-    __MAX_SENSOR_DIMENSION = 1e-3
-    __MIN_XY_SIZE_CUSHION = 5e-4
-    __MIN_Z_SIZE_CUSHION = 1e-3
-    #: Default flag value indicating whether to use user provided thermal parameters.
+    """Default sensor dimension (m)."""
+    MIN_SENSOR_DIMENSION = 1e-4
+    """Minimum sensor dimension (m)."""
+    MAX_SENSOR_DIMENSION = 1e-3
+    """Maximum sensor dimension (m)."""
+    MIN_XY_SIZE_CUSHION = 5e-4
+    """Minimum cushion between sensor dimension and sample size in X and Y dimensions (m)."""
```

Comment:
```suggestion
    """Minimum cushion between sensor dimension and sample size in the X and Y dimensions (m)."""
```
For coordinates, you used lower case x, y, z. Here, you are using upper case. Just pointing this out....